managed/ui/src/components/config/Security/KeyManagementConfiguration.js

```diff
@@ -285,6 +285,7 @@ class KeyManagementConfiguration extends Component {
     if (listView) {
       return (
         <ListKeyManagementConfigurations
+          {...this.props}
```

Comment:
do you really need to forward all parent props downstream?

Reply:
If we handle the state to toggle the modal locally then it's not needed.